nvd3/linePlusBarChart.py

```diff
@@ -104,17 +104,22 @@ def __init__(self, **kwargs):
                                         "function(d) { return d3.format(',f')(d) }")
         self.yaxis2_format = kwargs.get('yaxis2_format',
                                         "function(d) { return d3.format(',f')(d) }")
-
+        # kwargs.get('x_is_date', True) means :add key('x_is_date') in dict,or/and set a defualt value while it=None.return the value after modify
```

Comment:
No worries, we are here to help too. It's a very good job for a first PR on this project!